agents/plugins/apache_status

```diff
@@ -162,6 +176,16 @@ def main():
     if not servers:
         return 0
 
+    overwrite_conf = {'custom': config.get("CUSTOM_ADDRESS_OVERWRITE", {})}
```

Comment:
* Ein Name wie "instance_name_map" wäre besser als 'overwrite_conf'. Ist ja zum Teil nicht konfiguriert, sondern aus dem Zustand des Systems abgeleitet, wenn ich das richtig verstehe.
* Dieser ganze neue Block wäre in einer Funktion besser aufgehoben "get_instance_name_map" oder so.

Reply:
gerne umgebaut wie gewünscht